climada/engine/impact.py

```diff
@@ -246,7 +244,7 @@ def from_eih(cls, exposures, impfset, hazard,
                                  axis=1),
             crs = exposures.crs,
             unit = exposures.value_unit,
-            tot_value = exposures.affected_total_value(hazard),
+            tot_value = exposures._centroids_total_value(hazard),
```

Comment:
The linter complains about accessing a private method, and I agree. I think there is no harm in making this method public and adding a hint/warning to its documentation that it is there for legacy purposes and should not be used (by others)